content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -723,12 +723,7 @@ Only statically provisioned volumes are supported for alpha release. Administrat
 
 ## Volume Snapshot and Restore Volume from Snapshot Support
 
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
-
-Volume snapshot feature was added to support CSI Volume Plugins only. For details, see [volume snapshots](/docs/concepts/storage/volume-snapshots/).
-
-To enable support for restoring a volume from a volume snapshot data source, enable the
-`VolumeSnapshotDataSource` feature gate on the apiserver and controller-manager.
+Volume snapshot feature is graduated in the 1.20 release. It only supports out-of-tree CSI Volume Plugins as in-tree Volume Plugins are deprecated. See the doc [here](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md) for more information on deprecated volume plugins. For details about volume snapshots, see [here](/docs/concepts/storage/volume-snapshots/).
```

Comment:
Hi @xing-yang .
I suggest to add back and update the feature state shortcode.

```
{{< feature-state for_k8s_version="v1.20" state="stable" >}}
```

Some other edits to consider:
- Could the first sentence be removed?
- Is the heading for this section meaningful?
- Improve the text for the links. Possible edits:

Volume snapshots only support the out-of-tree CSI volume plugins. For details, see [Volume Snapshot](/docs/concepts/storage/volume-snapshots/).
In-tree Volume plugins are deprecated. You can read about the deprecated volume plugins in the [Volume plugin FAQ](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md).

Reply:
The heading mentions "Restore Volume from Snapshot Support" because there is a sub-section about it below.  
I have addressed other comments.  Thanks.